Makefile

```diff
@@ -191,10 +191,11 @@ docker:
 	chmod +x docker
 
 semaphore: docker
-	# Use the downloaded docker locally, not just with Docker in Docker STs
-	service docker stop
-	cp ./docker $(shell which docker)
-	service docker start
+	# Commenting out.
+	# See: https://github.com/projectcalico/calico-containers/pull/944
```

Comment:
Can we get rid of this.  If you do want to add something here, just copy the comment that you added to the semaphore section in calico-containers